packages/zwave-js/src/lib/commandclass/SceneActuatorConfigurationCC.ts

```diff
@@ -193,21 +192,52 @@ export class SceneActuatorConfigurationCCAPI extends CCAPI {
 		await this.driver.sendCommand(cc, this.commandOptions);
 	}
 
+	public async getActive(): Promise<
+		| Pick<
+				SceneActuatorConfigurationCCReport,
+				"sceneId" | "level" | "dimmingDuration"
+		  >
+		| undefined
+	> {
+		this.assertSupportsCommand(
+			SceneActuatorConfigurationCommand,
+			SceneActuatorConfigurationCommand.Get,
+		);
+
+		const cc = new SceneActuatorConfigurationCCGet(this.driver, {
+			nodeId: this.endpoint.nodeId,
+			endpoint: this.endpoint.index,
+			sceneId: 0,
+		});
+		const response = await this.driver.sendCommand<SceneActuatorConfigurationCCReport>(
+			cc,
+			this.commandOptions,
+		);
+
+		if (response) {
+			return pick(response, ["sceneId", "level", "dimmingDuration"]);
+		}
+	}
+
 	public async get(
 		sceneId: number,
 	): Promise<
-		| {
-				sceneId: number;
-				level: number;
-				dimmingDuration: Duration;
-		  }
+		| Pick<SceneActuatorConfigurationCCReport, "level" | "dimmingDuration">
 		| undefined
 	> {
 		this.assertSupportsCommand(
 			SceneActuatorConfigurationCommand,
 			SceneActuatorConfigurationCommand.Get,
 		);
 
+		if (sceneId === 0) {
+			// We need to prevent this, since it will cause the Report sceneId
+			// to be different from the requested sceneId, and the client won't
+			// be able to determine the actual sceneId, because it's hidden in
+			// the return value
+			return;
```

Comment:
Just throw here with error code `Argument_Invalid` and tell the dev to use `getActive`.